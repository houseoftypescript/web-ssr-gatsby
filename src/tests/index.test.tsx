import { render } from '@testing-library/react';
import { PageProps } from 'gatsby';
import React from 'react';
import IndexPage from '../pages';

describe('IndexPage', () => {
  it('to match snapshot', () => {
    const { container } = render(<IndexPage {...({} as PageProps)} />);
    expect(container).toMatchSnapshot();
  });
});
