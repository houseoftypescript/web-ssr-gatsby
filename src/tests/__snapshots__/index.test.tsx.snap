// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage to match snapshot 1`] = `
<div>
  <main
    class="w-full h-screen flex flex-col items-center justify-center gap-4"
  >
    <h1
      class="text-9xl font-bold uppercase"
    >
      Gatsby
    </h1>
    <p
      class="text-3xl"
    >
      TypeScript + Tailwind CSS + Material UI
    </p>
  </main>
</div>
`;
